app/views/layouts/_header.html.haml

```diff
@@ -5,8 +5,9 @@
         = link_to root_path do
```

Comment:
This also needs to consider `HackathonConfig['homepage_url']`

On second thought, maybe it's easier if we use `root_path` as a default? Something like adding this to the top of the file:

```haml
- homepage_url = HackathonConfig['homepage_url'].presence || root_path
```

...and then use this new local `homepage_url` variable in place of where `root_path` is currently used. Also would avoid the need for extra if/else logic that is complicating this